### PR TITLE
Development container environment

### PR DESCRIPTION
**Pull request by Gehock**
# Description

This pull request adds a dev/test environment similar to [run-aplus-front](https://github.com/apluslms/run-aplus-front)+[aplus-manual](https://github.com/apluslms/aplus-manual) using docker and docker-compose. The PR is marked as a draft because the feature is still very much work-in-progress and requires some clean-up before merging, but feedback and testing are already welcome at this stage. The first commit ("Add initial files from run-aplus-front") contains a squashed history of [run-aplus-front](https://github.com/apluslms/run-aplus-front/commit/b1c16b97c4bc4c7a5a9c1f99e7f304931683a5d7), on top of which this feature has been built.

## Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [ ] Aplus Manual.
- [x] Other documentation (mention below which documentation).

There is a separate documentation file at `docker/README.md`.

## Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

**Comment by Gehock** [comment]
Documentation still requires some small changes but otherwise this is more or less ready

**Comment by Gehock** [comment]
@Mankro This PR is more or less ready, I could use some small feedback on whether the documentation is understandable enough, but otherwise this is ready to merge if it looks good to you

**Review by Mankro — APPROVED** [review]
Excellent work! Shall we squash the git commits before merging?

File: docker/build.sh
Comment (Gehock) [reply docker/build.sh]: Changed the image name to `apluslms/run-radar`

File: docker/manual-services.md
Comment (Gehock) [reply docker/manual-services.md:2]: Those commands would work, added a small clarification to the beginning of the file.

File: docker/Dockerfile
Comment (Gehock) [reply docker/Dockerfile]: The `flower` package hasn't created an official release in some time so the version in [PyPi](https://pypi.org/project/flower/) is outdated. Clarified the comment a bit.

**Comment by Gehock** [comment]
> Shall we squash the git commits before merging?

I would prefer creating a merge commit instead. I tried to reorganise the commit history into a set of logical changes, so the commit history itself now acts as some documentation of the development process. If nothing else, I think it would be important to make a distinction between the first commit, which just contains a squashed version of [run-aplus-front](https://github.com/apluslms/run-aplus-front), and the rest of the commits that actually introduce the radar-specific functionality.

**Pull request merged**
